src/express/routes/articles-routes/articles-routes.js

```diff
@@ -4,22 +4,36 @@ const {Router} = require(`express`);
 
 const {getAPI} = require(`../../api`);
 const upload = require(`../../middlewares/upload`);
+const {ARTICLES_PER_PAGE} = require(`../../../constants`);
 
 const articlesRouter = new Router();
 const api = getAPI();
 
 articlesRouter.get(`/category/:id`, async (req, res) => {
-  const categories = await api.getCategories(true);
-  const selectedCategoryId = req.params.id;
-  const activeCategoryTab = categories.find((it) => it.id === parseInt(selectedCategoryId, 10));
+  let {page = 1} = req.query;
+
+  page = +page;
```

Comment:
в таком случае декомпозиция привносит больше сложности чем пользы, можно заменить на
 ```suggestion
  const page = +req?.query?.page || 1;
```